pkgs/tools/networking/nss-mdns/default.nix

```
@@ -1,14 +1,18 @@
-{ fetchurl, lib, stdenv, fetchpatch }:
+{ fetchurl, lib, pkgs, stdenv, fetchpatch, fetchFromGitHub }:
```

```suggestion
{ fetchurl, lib, autoreconfHook, pkg-config, stdenv, fetchpatch, fetchFromGitHub }:
```

Do not pass all of pkgs here, pass individual packages as required